crates/rpc/rpc/src/result.rs

```diff
@@ -105,7 +106,7 @@ impl_to_rpc_result!(reth_interfaces::RethError);
 impl_to_rpc_result!(reth_network_api::NetworkError);
 
 /// An extension to used to apply error conversions to various result types
-pub(crate) trait ToRpcResultExt {
+pub trait ToRpcResultExt {
```

Comment:
I'm okay with making this pub

Reply:
I'm not using this trait anymore actually. Instead I made the ToRpcResult trait public:
https://github.com/paradigmxyz/reth/pull/4783/commits/f672f19f46ad4413cbdee5455931306f0310ba62

Lmk if I should keep this one public as well or revert.